src/data/nat/succ_pred.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2021 Yaël Dillies. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yaël Dillies
 -/
+import data.fin.basic
```

Comment:
Just noting for later: this import is bogus. I remember adding the few `fin` lemmas at the bottom of the file because that was the first place where I had both access to `covby` and `fin` (due to the bogus import this PR is fixing), but those lemmas have nothing to do with `succ`/`pred` on `nat`.